pipelines/backup-apim-pipeline.yaml

```diff
@@ -0,0 +1,74 @@
+trigger: none
+
+schedules:
+- cron: "0 1 * * *"
+  displayName: Every day at 1 am.
+  branches:
+    include:
+    - master
+
+parameters:
+  - name: 'SUBSCRIPTION_SERVICE_CONNECTION'
+    displayName: 'Azure subscription service connection name hosting the infrastructure built with terraform'
+    type: string
+    default: PROD-IO-SERVICE-CONN
+    values:
+      - PROD-IO-SERVICE-CONN
+      - DEV-IO-SERVICE-CONN
+  - name: 'APIM_RG'
+    displayName: 'API Management resource group name.'
+    type: string
+    default: io-p-rg-internal
+  - name: 'APIM_NAME'
+    displayName: 'API Management name'
+    type: string
+    default: io-p-apim-api
+  - name: 'STORAGE_ACCOUNT_RG'
+    displayName: 'Storage account resource group where to store the backup.'
+    type: string
+    default: io-p-rg-operations
+  - name: 'STORAGE_ACCOUNT_NAME'
+    displayName: 'Storage account where to store the backup.'
+    type: string
+    default: iopstbackups
+  - name: 'CONTAINER_NAME'
+    displayName: 'Name of the container inside the storage account'
+    type: string
+    default: apimbackup
+
+pool:
+  vmImage: 'ubuntu-latest'
+
+jobs:
+  - job: backup_apim
+    steps:
+      # 1. Run power shell task to backup apim instance into a storage account.
+      - task: AzurePowerShell@5
```

Comment:
Why PowerShell and not bash?

Reply:
That's what the guys from the support suggested us in yesterday's meeting. It looks like it's more complicated with the az cli.
Last but not least I learned a bit of Power Shell :-) ... First time for me ...

Reply:
Idk Powershell good enough to say this is an issue as it is in bash, but be sure that the whole task fails as soon as one of its instructions does (that is, what in bash we prevent by using `set -e` modifier)

Reply:
hi @balanza @uolter , you can change the FailOnStandardError (default: false) task parameter to accomplish the objective (https://docs.microsoft.com/en-us/azure/devops/pipelines/tasks/deploy/azure-powershell?view=azure-devops#yaml-snippet)